### ISSUE TITLE
phone_number_field different between PersonalInformation and DGFIP scopes

### ISSUE THREAD

**Issue by giffarda** [fabmob/CMS#10]
https://github.com/fabmob/CMS/blob/110ac7e2aa9985e571f2423cf35ee24cb08d241d/conceptual_model/schemas/personal-information.schema.json#L146-L150

https://github.com/fabmob/CMS/blob/110ac7e2aa9985e571f2423cf35ee24cb08d241d/conceptual_model/schemas/fr-dgfip-information.schema.json#L276-L279

Hi, 

For phone number field type, a boolean "mobile" was added. It is indicated in the [DGFIP schema.json](https://github.com/fabmob/CMS/blob/main/conceptual_model/schemas/fr-dgfip-information.schema.json) but not in [PersonalInformation schema.json](https://github.com/fabmob/CMS/blob/main/conceptual_model/schemas/personal-information.schema.json). Can you confirm we want this boolean in all scopes which need a phone number ?

Thanks in advance

**Comment by jthiard** [comment]
There seems to be a mismatch between the json schemas and the markdown documentation for that matter.
 
To move this forward I suggest that we add this `mobile` property in all places where a phone number is expected, but that we make this property optional (it is required in the DGFIP scope at the moment).

Does this will fit your use case ?

**Comment by giffarda** [comment]
Yes, it is good for us. In my opinion, the important thing is that the phone number type is the same in all over the specification, as much as possible. 
The property _mobile_ can be required or optional, but _optional_ seems to be a right choice.
